ui-spacetimechart/src/components/OccupancyBlockLayer.tsx

```diff
@@ -0,0 +1,45 @@
+import { useCallback } from 'react';
+
+import { useDraw } from '../hooks/useCanvas';
+import { type DrawingFunction } from '../lib/types';
+
+// Should we keep it ?
+export enum SignalType {
```

Comment:
If it's unused, I'd say nuke it - we can reintroduce it easily if/when we need it.

Reply:
good point, done